.changeset/config.json

```diff
@@ -10,5 +10,5 @@
   "access": "restricted",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": ["@sample/*"]
```

Comment:
I think if the sample are private, this will not be necessary

Reply:
With squide I had to add it even after I made the packages private, otherwise, they still showed up in the changeset menu.